stubs/cachetools/cachetools/__init__.pyi

```diff
@@ -45,18 +49,35 @@ class RRCache(Cache[_KT, _VT]):
     @property
     def choice(self) -> Callable[[Sequence[_KT]], _KT]: ...
 
-class TTLCache(Cache[_KT, _VT]):
+class _TimedCache(Cache[_KT, _VT]):
     def __init__(
-        self, maxsize: float, ttl: float, timer: Callable[[], float] = ..., getsizeof: Callable[[_VT], float] | None = ...
+        self, maxsize: float, timer: Callable[[], float] = ..., getsizeof: Callable[[_VT], float] | None = ...
     ) -> None: ...
     @property
     def currsize(self) -> float: ...
     @property
```

Comment:
```suggestion
    class _Timer:
        def __init__(self, timer: Callable[[], float]) -> None: ...
        def __call__(self) -> float: ...
        def __enter__(self) -> float: ...
        def __exit__(self, *exc: object) -> None: ...
    @property
```

And then just return `_Timer` from the `timer()` method below?

Reply:
Let's see what CI says about nested classes.